lib/jekyll-import/importers/tumblr.rb

```diff
@@ -131,8 +131,10 @@ def self.post_to_hash(post, format)
           when "video"
             title = post["video-title"]
             content = post["video-player"]
-            unless post["video-caption"].nil?
-              content << "<br/>" + post["video-caption"]
+            unless post[:content].nil?
```

Comment:
Would you like me to make this change to the PR? Unfortunately, I would have no way to try it out though...